helm/kube-image-keeper/templates/registry-statefulset.yaml

```diff
@@ -95,7 +95,7 @@ spec:
 {{- if .Values.registry.persistence.enabled }}
   volumeClaimTemplates:
   - metadata:
-      name: {{ include "kube-image-keeper.fullname" . }}-data
+      name: data
```

Comment:
PVC names are scopped per namespace and it may be confusing and too generic to name it `data` (especially if kuik is installed in a mutualised namespace like kube-system). I guess it will be better to keep the prefix with `kube-image-keeper.fullname`.

Reply:
The name of a PVC created by a volumeClaimTemplate is actually postfixed with the statefulset name and id, prefixing it with the fullname would be redundant.

```bash
$ kubectl get sts -n kuik-system kuik-registry -o yaml | yq '.spec.volumeClaimTemplates[0].metadata.name'
data

$ kubectl get pvc -n kuik-system
NAME                       STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS       AGE
data-kuik-registry-0       Bound    pvc-a2f3fb58-8888-4ca3-8618-edd8975d0ce9   5Gi        RWO            openebs-hostpath   45h
```

Reply:
Ok i didn't catch that 👍